PutAndVerify/MC.tla

```diff
@@ -0,0 +1,21 @@
+---- MODULE MC ----
+EXTENDS WriteAndVerify, TLC
+
+\* MV CONSTANT declarations@modelParameterConstants
+CONSTANTS
+Alice, Bob
+----
+
+\* MV CONSTANT definitions CLIENTS
+const_171964209906974000 == 
```

Comment:
this TLA spec is not heavy, can you add more clients?

Reply:
I run it with 3 clients and no error reported. Also if someone wants to verify with more clients, he may modify the model constants manually.

I prefer keeping two clients by default because the state transform graph can still be read by a human... Starting from 3 the state transform gets too complex to be read...